OpenRA.Game/Input/Hotkey.cs

```diff
@@ -28,7 +28,12 @@ public static bool TryParse(string s, out Hotkey result)
 
 			Keycode key;
 			if (!Enum<Keycode>.TryParse(parts[0], true, out key))
-				return false;
+			{
+				int c;
```

Comment:
`int c` was indeed the correct thing here.  The value is being initialized by the TryParse in the next line.